### PR TITLE
HRINT-365 Java Demo - Edit Document

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/EditDocument.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+public class EditDocument {
+
+    public void run(RunParams runParams) {
+        String companyName = runParams.getCompanyName();
+
+        //region Demo
+        try (IDocumentSession session = DocumentStoreHolder.store.openSession()) {
+            //region Step_1
+            Company company = session.load(Company.class,"companies/5-A");
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/EditDocument.java]: please put space after comma

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/metadata.json [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    ],
+
+    "Walkthroughs": [
+        {
+            "Title": "Load an existing document",
+            "Slug" : "step-1",
+            "DescriptionLines": [
+                "* Load an existing document from the Document Store using the Session `load()` command.<br>",
+                "",
+                "* The loaded document is added to the internal entities map (dictionary) that the Session manages.<br>",
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/metadata.json]: dictionary -> map. 

In Java we use maps, in c# this data structure is called Dictionary

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/EditDocument.java (hunk 1 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/EditDocument.java]: done

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/metadata.json (hunk 2 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/metadata.json]: so will it be good just to say the following?

"The loaded document is added to the internal entities map that the Session" manages

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/deleteDocument/DeleteDocument.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+            //region Step_2
+            session.saveChanges();
+            //endregion
+        }
+        //endregion
+    }
+
+    public static class RunParams {
+        private String DocumentId;
```
Comment (ml054) [inline DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/deleteDocument/DeleteDocument.java]: fields starts with lower case in java

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/metadata.json (hunk 2 above)
Comment (ml054) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/editDocument/metadata.json]: I think just rename Dictionary to Map.

File: DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/deleteDocument/DeleteDocument.java (hunk 3 above)
Comment (Danielle9897) [reply DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/basics/deleteDocument/DeleteDocument.java]: done

**Comment by Danielle9897** [comment]
Pls note: PR is actually for issue **HRINT-356** and not **365** ...

**Pull request merged**